rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_participant_info.hpp

```diff
@@ -76,6 +76,7 @@ class CustomTopicListener final : public eprosima::fastdds::dds::TopicListener
   void remove_event_listener(EventListenerInterface * event_listener);
 
 private:
+  std::mutex event_listeners_mutex_;
```

Comment:
nit: I guess we now should add `RCPPUTILS_TSA_GUARDED_BY` to `event_listeners_`

Reply:
Good call, fixed in 8e73176203e41ca355636163f54b5b6f6915345b.